### ISSUE TITLE
Give an 8 to 10 second time frame for adding a new student

### ISSUE THREAD

**Issue by Vibhu-Agarwal** [Marauders-9998/Attendance-Management-using-Face-Recognition#4]
A relatively easier counterpart of issue #3 
While adding a new student, currently the app asks it's users to get their 5 photos captured.
I want to reduce this step which is not user-friendly because student has to get the photo captured and confirm again and again.
A good solution to this could be to give an 8 to 10 second time frame to student to move around and internally, the app can capture 40-50 photos.
This way not only the redundant steps of manually capturing the photo will be removed, but more photos will be captured and thus a better training data will be obtained right from the starting.

**Comment by NiharikaVadlamudi** [comment]
So , essentially you want to capture video , and then sample (40-50) them into image frames ? (If this is what you are looking for , then I can work on this , if you assign  it to me)

**Comment by Vibhu-Agarwal** [comment]
> So , essentially you want to capture video , and then sample (40-50) them into image frames ?

Yes. But we don't want to completely replace the current implementation.
What we need is an option which can be passed as a flag to the application so that it's able to use different implementations.

> I do need few clarifications on some parts

Shoot away your doubts.

> Can this be assigned to me ?

Sure, but please put #29 higher up your priority list :)
